autometa/binning/recursive_dbscan.py

```diff
@@ -30,6 +30,7 @@
 
 from autometa.common.markers import Markers
 from autometa.common import kmers
+from autometa.common.exceptions import RecursiveDBSCANError
 # TODO: This should be
 # from autometa.common.kmers import Kmers
 # So later we can simply/and more clearly do Kmers.load(kmers_fpath).embed(method)
```

Comment:
This comment is now in the wrong place.

Reply:
I'm not sure I understand the comment. Do you mean to mark this as resolved to avoid confusion?

Reply:
Where it says `TODO: This should be from autometa.common.kmers import Kmers`. The comment refers to line 32, and so is in the wrong place because you inserted line 33 in there.

Reply:
Ah, I see. I'll fix this comment.